components/Accordion/Accordion.jsx

```diff
@@ -20,67 +21,82 @@ const Wrapper = styled.ul`
       },
     },
   }) => `
+    color: ${color};
     font-size: ${baseFontSize}px;
     padding: 0 ${medium}px;
-    color: ${color};
   `}
-  list-style-type: none;
   background-color: inherit;
+  list-style-type: none;
 `;
 
-const AccordionItem = styled.li`
-  border-bottom: 1px solid
-    ${({ opened, color }) => (opened ? 'transparent' : color)};
-
-  &:last-child {
-    border-bottom: 1px solid transparent;
-  }
-`;
+const AccordionItem = styled.li``;
```

Comment:
is it need this?

Reply:
AccordionItem is needed to find it self in unit test.
If i remove it, will be necessary to put a class in html renderized and find in unit test by class ( in css tree way).
I believe keeping in this way, make it more clear and easier to maintain.
And, i guess, faster too.

What do you think?

Reply:
Can you use the parents to selector element in unit test? Use styled-components to create element without properties it's wrong for me. 🍡

Reply:
Agree. I fixed changed the styled element to a css class.